checks.go

```diff
@@ -102,19 +103,23 @@ func runCheck(cond cond) bool {
 	debug("Running condition:\n", cond)
 
 	not := "Not"
+	regex := "Regex"
 	condFunc := ""
 	negation := false
+	cond.Regex = false
```

Comment:
Setting condition regex as a boolean works since it's more nuanced than negation, although I think we should leave it unexported (like starting with a lowercase in the struct) which should prevent people from trying to set it in their config. That or raise an error, since if they have Regex = true but `CommandContains` not `CommandContainsRegex` then it won't be regex.